utils/gnomad_functions.py

```diff
@@ -90,6 +90,27 @@ def add_variant_type(alt_alleles: hl.expr.ArrayExpression) -> hl.expr.StructExpr
     ), n_alt_alleles=hl.len(non_star_alleles))
 
 
+def adjusted_sex_ploidy_expr(
+        locus_expr: hl.expr.LocusExpression,
+        gt_expr: hl.expr.CallExpression,
+        sex_expr: hl.expr.StringExpression,
+        male_str: str = 'male',
```

Comment:
should we change these (`male_str`) and (`female_str`) to the karyotypes (XY, XX)?